docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-create-resources-with-parent-child-relationships.md

```diff
@@ -96,4 +104,4 @@ class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefro
 }
 ```
 
-If everything is set up correctly, you should be able to access `http://glue-storefront.mysprykershop.com/foo/1/bar` .
+If everything is set up correctly, you can access `https://glue-storefront.mysprykershop.com/foo/1/bar`.
```

Comment:
We must avoid using "should" - https://developers.google.com/style/word-list#should
```suggestion
Now, you can access `https://glue-storefront.mysprykershop.com/foo/1/bar`.
```